src/geop/services/src/main/scala/com/azavea/mmw/GeopServiceActor.scala

```diff
@@ -0,0 +1,69 @@
+package com.azavea.mmw
+
+import akka.actor.Actor
+import spray.routing._
+import spray.http._
+
+object UnattractiveButTemporary {
+  import geotrellis.raster.io.geotiff.reader._
+  import geotrellis.raster._
+  import geotrellis.raster.reproject._
+  import geotrellis.vector._
+  import geotrellis.vector.reproject._
+  import geotrellis.vector.io.json._
+  import geotrellis.spark.utils.SparkUtils
+  import geotrellis.spark._
+  import geotrellis.spark.tiling._
+  import geotrellis.raster.mosaic._
+  import geotrellis.proj4._
+
+  import org.apache.spark._
+  import org.apache.spark.{SparkConf, SparkContext}
+
+  import spray.json._
+  import spray.json.DefaultJsonProtocol._
+  import spray.httpx.unmarshalling._
+
+  import DataFiles._
+  import NonHistogram._
+
+  implicit val sc: SparkContext = SparkUtils.createSparkContext("Geoprocessing", new SparkConf())
+  val nlcd = DataFiles.delawareNLCD
+  val soil = DataFiles.delawareSoil
+
+  def jsonHistogram(str: String) : String = {
+    val polygon = str.parseJson.convertTo[Polygon]
+    val histogram: List[((Int, Int), Int)] = NonHistogram.gtHistogram(nlcd, soil, polygon).toList
+    histogram.toJson.prettyPrint
+  }
+}
+
+class GeopServiceActor extends Actor with GeopService {
+  def actorRefFactory = context
+  def receive = runRoute(myRoute)
+}
+
+trait GeopService extends HttpService {
+  import scala.util._
+
+  import MediaTypes._
+  import UnattractiveButTemporary._
+
+  val myRoute =
```

Comment:
Is this setup to be a shortcut or the ultimate mechanism to submit jobs to GeoTrellis? It seems like we might benefit from using [`spark-jobserver`](https://github.com/spark-jobserver/spark-jobserver) here instead.